### PR TITLE
arch-update: add support for yay AUR helper

### PR DESCRIPTION
**Pull request by Q-Mart**

### CODE REVIEW

File: arch-update/arch-update [hunk 1]
```diff
@@ -37,10 +39,7 @@ def create_argparse():
     parser.add_argument(
         '-a',
         '--aur',
-        action = 'store_const',
-        const = True,
-        default = _default('AUR', 'False', strbool),
-        help='Include AUR packages. Attn: Yaourt must be installed'
```
Comment (jolange) [inline arch-update/arch-update]: The configuration should be done using environment variables (see #132). The arguments are just kept for backwards compatibility. Also, please ensure that the functionality stays backwards compatible: `AUR=true` should still work.

**Comment by jolange** [comment]
@Q-Mart any updates? Or should we close this?

**Comment by Q-Mart** [comment]
No updates, feel free to close

**Comment by jolange** [comment]
Ok, thanks for the feedback!

**Pull request closed without merging**